.github/workflows/helm.yaml

```diff
@@ -0,0 +1,32 @@
+name: Publish Helm artifact
+
+on:
+  workflow_dispatch:
```

Comment:
we could add a `[on.release](https://docs.github.com/en/actions/writing-workflows/choosing-when-your-workflow-runs/events-that-trigger-workflows#release)`, right? then it would be triggered automatically.

```
on:
  release:
    types: [published]
```

Reply:
@dgn 
Not exactly.

As I mentioned in the comment above, it depends on the type of GITHUB TOKEN being used.
I've added link, which explains that if the workflow in using the default GITHUB TOKEN, which is being generated by the workflow for each execution and the workflow itself is using a `workflow_dispatch` jobs type, the other workflow job will not be triggered even if it set to check the published release.

I've tested it and it indeed does not work.

In the same github docs, there is an explanation that it could be workarounded by using a custom create GITHUB TOKEN.
And I'm going to test it.
But I prefer to add the existing workflow with the manual trigger as per now, to provide the option to publish the helm artifacts to helm repo.

Will check the complete automation flow as a next step.

Reply:
ah. we already have a GIT_TOKEN secret that you could use. it's the token of our bot account, openshift-service-mesh-bot

Reply:
Yes, I know that we're using a custom token.
I saw it.

I just need to catch Brian and ask him regarding exact config of this token.
So I could simulate and properly test its flow.